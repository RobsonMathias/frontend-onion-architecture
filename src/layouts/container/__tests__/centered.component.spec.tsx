/**
 * @jest-environment jsdom
 */
import '@testing-library/jest-dom/extend-expect';

import { render } from '@testing-library/react';
import React from 'react';

import { CenteredComponent } from '../centered.component';

describe('CenteredComponent', () => {
  const defaultProps = {};

  it('should render successfully', () => {
    const { container } = render(
      <CenteredComponent {...defaultProps}>
        <p>Working!</p>
      </CenteredComponent>
    );
    expect(container).toMatchSnapshot();
  });
});
