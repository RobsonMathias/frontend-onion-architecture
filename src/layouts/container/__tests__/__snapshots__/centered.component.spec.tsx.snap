// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CenteredComponent should render successfully 1`] = `
<div>
  <div
    class="MuiBox-root css-36s324"
  >
    <div
      class="MuiGrid-root MuiGrid-container css-1vam7s3-MuiGrid-root"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthXs css-nzinn5-MuiContainer-root"
      >
        <p>
          Working!
        </p>
      </div>
    </div>
  </div>
</div>
`;
